R/PackageFunctionReporter.R

```diff
@@ -73,21 +79,24 @@ PackageFunctionReporter <- R6::R6Class(
             # Avoid mvbutils::foodweb bug on one function packages
             numFuncs <- as.character(unlist(utils::lsf.str(asNamespace(private$packageName)))) # list of functions within Package
             if (length(numFuncs) == 1) {
-                log_warn(sprintf('No Network Available.  Only one function in %s.', private$packageName))
-                
-                nodeDT <- data.table::data.table(nodes = numFuncs, level = 1,  horizontal = 0.5)
-                return(packageObj <- list(nodes = nodeDT, edges = list(), networkMeasures = list()))
+                log_info("Only one function. Edge list is null.")
+                return(NULL)
             }
             
             log_info(sprintf('Constructing network representation...'))
             funcMap <- mvbutils::foodweb(where = paste("package", private$packageName, sep = ":"), plotting = FALSE)
             log_info("Done constructing network representation")
             
-            # Function Connections: Arcs
+            # Function Connections: Edges
             edges <- data.table::melt(data.table::as.data.table(funcMap$funmat, keep.rownames = TRUE)
                                       , id.vars = "rn")[value != 0]
             data.table::setnames(edges,c('rn','variable'), c('TARGET','SOURCE'))
             
+            # If no edges, return NULL
+            if (nrow(edges) == 0) {
+              return(NULL)
```

Comment:
can you make this `return(invisible(NULL))`? Nobody likes getting random NULLs printed to their logs